<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServeurWidget</class>
 <widget class="QWidget" name="ServeurWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>332</width>
    <height>315</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ServeurWidget</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Num√©ro de port</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxPort">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>30</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maximum">
    <number>9999</number>
   </property>
   <property name="value">
    <number>1025</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonLancementServeur">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>271</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Lancement serveur</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Clients</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>271</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonQuitter">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>271</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonQuitter</sender>
   <signal>clicked()</signal>
   <receiver>ServeurWidget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>204</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>292</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
